docs/RBC_EN.md

```diff
@@ -0,0 +1,52 @@
+# RBC : Reliable Broadcast
+
+
+
+### overview
+
+The typical Broadcast, which depends on implementation, does not guarantee message delivery and does not consider situations in which the sender is crash (the situation in which the message cannot be sent to all). However, the RBC guarantees message delivery and partially solves the situation where the sender crashes. And the Atomic Broadcast introduced in this [paper](<https://eprint.iacr.org/2016/199.pdf>) is a Broadcast that guarantees the situation of the sender crash and guarantees the order of messages, and the RBC to be introduced is Atomic Broadcast.
```

Comment:
The Atomic Broadcast seems to be suddenly introduced, so I suggest this as follow!

```suggestion
The typical Broadcast, which depends on implementation, does not guarantee message delivery and does not consider situations in which the sender is crash (the situation in which the message cannot be sent to all). However, the RBC guarantees message delivery and partially solves the situation where the sender crashes. 



In this [paper](<https://eprint.iacr.org/2016/199.pdf>), the Atomic Broadcast is introduced. This is a Broadcast that guarantees the situation of the sender crash and the order of messages. And this Atomic Broadcast is called the **RBC**.
```